### ISSUE TITLE
Bug in pair.compute_current_rdf()

### ISSUE THREAD

**Issue by tcmoore3** [ctk3b/msibi#15]
We added the max_frames argument to help with loading large trajectories that take up too much memory. However, the normalizing is done incorrectly.

       for last_frame in range(int(max_frames), 
                int(state.traj.n_frames + max_frames), int(max_frames)):
            r, g_r = md.compute_rdf(state.traj[first_frame:last_frame],
                    pairs, r_range=r_range / 10, n_bins=n_bins)
            if g_r_all == None:
                g_r_all = np.zeros_like(g_r)
            g_r_all += g_r / len(state.traj[first_frame:last_frame])
            first_frame = last_frame

The line 
`g_r_all += g_r / len(state.traj[first_frame:last_frame])`
is where the issue is. The RDFs are divided by the number of frames in each chunk, which is making them appear smaller than they actually are.

**Comment by ctk3b** [comment]
Should we instead by writing the following?
```python
g_r_all += g_r * len(state.traj[first_frame:last_frame]) / total_frames
```

**Comment by tcmoore3** [comment]
Fixed in commit `7dcff06a7f5771fc54a7c8dbd0e03be834973896`
